src/Shared/Recruit.Vacancies.Client/Domain/Entities/UserNotificationPreferences.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
+{
+    public class UserNotificationPreferences
+    {
+        // user's id
+        public Guid Id { get; set; }
+        public NotificationTypes NotificationTypes { get; set; }
```

Comment:
I have merely followed the design document. However, I figured what would the queries look like in each case:
```
db.getCollection('userNotificationPreferences').find({notificationTypes:/VacancyRejected/ })
db.getCollection('userNotificationPreferences').find({notificationTypesArr:"VacancyRejected" })
```
First one is contains query in a string field and the following one is contains query in array.

There is very subtle difference, but it would be interesting to find out when we have thousands of users.

Reply:
Bear in mind I think this will likely change 🤣